### PR TITLE
[Core] Add order token length to configuration

### PR DESCRIPTION
**Pull request by NoResponseMate**
| Q               | A
|-----------------|-----
| Branch?         | 1.12
| Bug fix?        | yes
| New feature?    | kinda
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | -
| License         | MIT

### CODE REVIEW

**Review by Wojdylak — DISMISSED** [review]

**Review by mpysiak — DISMISSED** [review]

File: src/Sylius/Bundle/ApiBundle/CommandHandler/Cart/PickupCartHandler.php [hunk 1]
```diff
@@ -37,6 +37,7 @@ public function __construct(
         private ObjectManager $orderManager,
         private RandomnessGeneratorInterface $generator,
         private CustomerRepositoryInterface $customerRepository,
+        private int $tokenLength = 10,
```
Comment (GSadee) [inline src/Sylius/Bundle/ApiBundle/CommandHandler/Cart/PickupCartHandler.php]: Why not `64` by default here as well?
```suggestion
        private int $tokenLength = 64,
```

File: src/Sylius/Bundle/CoreBundle/DependencyInjection/Configuration.php [hunk 2]
```diff
@@ -54,6 +54,10 @@ public function getConfigTreeBuilder(): TreeBuilder
                     ->setDeprecated('sylius/sylius', '1.10', 'The "%path%.%node%" parameter is deprecated and will be removed in 2.0.')
                     ->defaultFalse()
                 ->end()
+                ->integerNode('order_token_length')
+                    ->defaultValue(64)
+                    ->min(1)->max(255)
```
Comment (GSadee) [inline src/Sylius/Bundle/CoreBundle/DependencyInjection/Configuration.php:59]: At least `10` as min value, wdyt?
```suggestion
                    ->min(10)->max(255)
```

**Review by GSadee — DISMISSED** [review]

File: src/Sylius/Bundle/ApiBundle/CommandHandler/Cart/PickupCartHandler.php (hunk 1 above)
Comment (diimpp) [reply src/Sylius/Bundle/ApiBundle/CommandHandler/Cart/PickupCartHandler.php]: I see new parameter `sylius_core.order_token_length` introduced, but I don't see any usage of it.

It needs to be injected to `Sylius\Bundle\ApiBundle\CommandHandler\Cart\PickupCartHandler` and `Sylius\Component\Core\TokenAssigner\UniqueIdBasedOrderTokenAssigner` services?

Comment (NoResponseMate) [reply src/Sylius/Bundle/ApiBundle/CommandHandler/Cart/PickupCartHandler.php]: Yeah, the commit setting these apparently poofed somehow.

**Review by TheMilek — APPROVED** [review]

**Comment by TheMilek** [comment]
Thank you, @NoResponseMate!

**Pull request merged**